python/ray/worker.py

```diff
@@ -1082,7 +1085,12 @@ def listen_error_messages_raylet(worker, task_error_queue, threads_stopped):
     worker.error_message_pubsub_client.psubscribe(error_pubsub_channel)
 
     try:
-        # Get the errors that occurred before the call to subscribe.
+        if _internal_kv_initialized():
```

Comment:
Btw, is this creating a duplicated connection or sharing the existing one?

Reply:
We shouldn't need to check initialized right?

Reply:
It's just a worker mode check, and all following calls should reuse the single Redis client on the worker.